cypress/e2e/models/migration/applicationinventory/analysis.ts

```diff
@@ -170,7 +172,22 @@ export class Analysis extends Application {
 
     protected selectTarget(target: string[]): void {
         for (let i = 0; i < target.length; i++) {
-            cy.get("div.pf-v5-c-empty-state__content").children("h4").contains(target[i]).click();
+            if (
+                target.includes("OpenJDK 11") ||
+                target.includes("OpenJDK 17") ||
+                target.includes("OpenJDK 21")
+            ) {
```

Comment:
Here we are inside of a for loop because `targets` is an array (we can select multiple target), so we have to use `target[i]` instead of the whole array.

Knowing that, we can also simplify the if condition by grouping the 3 targets we want to check

```typescript
if(['OpenJDK 11', 'OpenJDK 17', 'OpenJDK 21']).includes(target[i]))
```

Reply:
Thanks Alex, added